input/fsh/Profiles clinical datapoints/TumorStaging.fsh

```diff
@@ -0,0 +1,184 @@
+Target: "https://connect.ichom.org/patient-centered-outcome-measures/breast-cancer"
+Id: ClinicalTNMTumorStageMapping
+Title: "Clinical TNM tumor stage to ICHOM set"
+Description: "Mapping of the clinical TNM tumor stage to the ICHOM breast cancer PCOM set." 	
+* valueCodeableConcept -> "Clinical tumor stage"
+
+// PATHOLOGICAL TUMOR STAGE
+Profile:  PathologicalTNMTumorStage
+Id: pathological-tnm-tumor-stage
+Parent: Observation
+Title: "Pathological TNM tumor stage"
+Description: "Represents the pathological stage of the tumor (per AJCC 8th Ed.)"
+* insert PublicationProfileRuleset
+* code = SCT#385356007 "Tumor stage finding"
+* subject only Reference(BreastCancerPatient)
+* value[x] only CodeableConcept 
+* value[x] from TNMPrimaryTumorVS (preferred)
+* value[x] MS
+* effectiveDateTime MS
+
+Instance: PathologicalTNMTumorStagePatient147
+InstanceOf: PathologicalTNMTumorStage
+Description: "Example showing TNM staging for tumor stage"
+* status = #final "final"
+* subject = Reference(BreastCancerPatient147)
+* valueCodeableConcept = AJCC#pTis
+* method = SCT#897275008 "American Joint Commission on Cancer, Cancer Staging Manual, 8th edition neoplasm staging system (tumor staging)"
+
+
+Mapping: PathologicalTNMTumorStageToICHOM
+Source:	PathologicalTNMTumorStage
+Target: "https://connect.ichom.org/patient-centered-outcome-measures/breast-cancer"
+Id: PathologicalTNMTumorStageMapping
+Title: "Pathological TNM tumor stage to ICHOM set"
+Description: "Mapping of the pathological TNM tumor stage to the ICHOM breast cancer PCOM set." 	
+* valueCodeableConcept -> "Pathological tumor stage"
+
+
+// CLINICAL NODAL STAGE
+Profile: ClinicalTNMNodalStage
+Id: clinical-tnm-nodal-stage
+Parent: Observation
+Title: "Clinical TNM nodal stage"
+Description: "Represents the clinical stage of the presence or absence of metastases in regional lymph nodes (per AJCC 8th Ed.)"
+* insert PublicationProfileRuleset
+* code = SCT#385382003 "nodal stage finding"
+* subject only Reference(BreastCancerPatient)
+* value[x] only CodeableConcept 
+* value[x] from TNMRegionalNodesVS (preferred)
+* value[x] MS
+* effectiveDateTime MS
+
+Instance: ClinicalTNMNodalStagePatient147
+InstanceOf: ClinicalTNMNodalStage
+Description: "Example showing clinical TNM staging for nodal stage"
+* status = #final "final"
+* subject = Reference(BreastCancerPatient147)
+* valueCodeableConcept = AJCC#cN1
+* method = SCT#897275008 "American Joint Commission on Cancer, Cancer Staging Manual, 8th edition neoplasm staging system (tumor staging)"
+
+Mapping: ClinicalTNMNodalStageToICHOM
+Source:	ClinicalTNMNodalStage
+Target: "https://connect.ichom.org/patient-centered-outcome-measures/breast-cancer"
+Id: ClinicalTNMNodalStageMapping
+Title: "Clinical TNM nodal stage to ICHOM set"
+Description: "Mapping of the clinical TNM nodal stage to the ICHOM breast cancer PCOM set." 	
+* valueCodeableConcept -> "Clinical nodal stage"
+
+
+// PATHOLOGICAL NODAL STAGE
+Profile: PathologicalTNMNodalStage
```

Comment:
This one looks a copy of ClinicalTNMNodalStage profile. Could we have only one profile?